lib/ProgramOptions/IniFileParser.h

```diff
@@ -68,24 +69,17 @@ class IniFileParser {
           "unable to open configuration file: no configuration file specified");
     }
 
-    std::ifstream ifs(filename, std::ifstream::in);
-
-    if (!ifs.is_open()) {
-      return _options->fail("unable to open configuration file '" + filename +
-                            "'");
-    }
-
+    auto buf = arangodb::basics::FileUtils::slurp(filename);
```

Comment:
this will now behave differently than the old implementation.
`slurp` will throw an exception, whereas the old implementation produced a much better error message.